packages/forms/src/elements/MultiThumbRange.spec.tsx

```diff
@@ -57,11 +67,103 @@ describe('MultiThumbRange', () => {
     });
   });
 
+  describe('MultiThumbRange label', () => {
+    test('clicking the label focuses the min thumb', () => {
```

Comment:
We don't use the `test()` helper in other test files. Let's stick with our standard `it()` usage.

Reply:
Opps, old habit. Will push new commit that uses `it` shortly.